internal/model/tabs/jobhisttab/jobhisttabview.go

```diff
@@ -95,36 +95,50 @@ func (jh *JobHistTab) View(l *log.Logger) string {
 		MainWindow strings.Builder
 	)
 
+	// Show parameters to the user
+	heading := lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("#FAFAFA")).Background(lipgloss.Color("#37ABBF")).Width(12).PaddingLeft(1).PaddingRight(1).Align(lipgloss.Right)
```

Comment:
Can we keep the fg `styles.Yellow` and bg `styles.Blue` as color scheme?
Usually wouldn't care, but the whole coloring scheme was picked on purpose for this one.

Reply:
I only now just reads through styles.go, I've gone for the TextYellowOnBlue style.